src/chainparams.cpp

```diff
@@ -247,17 +248,19 @@ class CTestNetParams : public CChainParams {
                 {336000, uint256S("0xf07adf61615c529f7c282b858d13d3e037b197324cb12e0669c461947494c4e3")},
                 {372751, uint256S("0x000000000000148db599b217c117b5104f5043c55f6ca2a8a065d9fab9f9bba1")},
                 {382019, uint256S("0x3ab75769d7957d9bf0857b5019d0a0e41044fa9ecf30b2f9c32aa457b0864ce5")},
+                {408500, uint256S("0x1636ac08b073d26b28fa40243d58dd5deb215752efe094c92c61998e4e9baf3f")},
```

Comment:
hmm, why not add a new block for checkpoint?

Reply:
Because this is where the SegWit fork happened.